README.md

```diff
@@ -252,18 +260,6 @@ _13 September 2024_
 - 🐛 Fixes some bugs building Linux wheels on macOS. (#1961)
 - ⚠️ Changes the minimum version of Docker/Podman to Docker API version 1.43, Podman API version 3. The only mainstream runner this should affect is Travis Graviton2 runners - if so you can [upgrade your version of Docker](https://github.com/pypa/cibuildwheel/pull/1961#issuecomment-2304060019). (#1961)
 
-
-### v2.20.0
-
-_4 August 2024_
-
-- 🌟 CPython 3.13 wheels are now built by default - without the `CIBW_PRERELEASE_PYTHONS` flag. It's time to build and upload these wheels to PyPI! This release includes CPython 3.13.0rc1, which is guaranteed to be ABI compatible with the final release. Free-threading is still behind a flag/config option. (#1950)
-- ✨ Provide a `CIBW_ALLOW_EMPTY` environment variable as an alternative to the command line flag. (#1937)
-- 🐛 Don't use uv on PyPy3.8 on Windows, it stopped working starting in 0.2.25. Note that PyPy 3.8 is EoL. (#1868)
-- 🛠 Set the `VSCMD_ARG_TGT_ARCH` variable based on target arch. (#1876)
-- 🛠 Undo cleaner output on pytest 8-8.2 now that 8.3 is out. (#1943)
-- 📚 Update examples to use Python 3.12 on host  (cibuildwheel will require Python 3.11+ on the host machine starting in October 2024) (#1919)
```

Comment:
Looks like this should be reverted...

Reply:
🤔 Yes, I think this came from the pre-commit hook. I'll remove it. I don't know if there's an alternative way to propose CHANGELOG entries.

Reply:
Removed and added it in a "Suggested CHANGELOG entry" section in the PR description. Could you please review the implementation once you have time? This is the best one that came to mind, but I'm happy to re-do any parts that aren't done well.